h/migrations/versions/6df1c8c3e423_revert_annotation_user_id.py

```diff
@@ -0,0 +1,18 @@
+"""Revert annotation.user_id."""
+from alembic import op
+
+revision = "6df1c8c3e423"
+down_revision = "076ac6c7a8e0"
+
+
+def upgrade():
+    op.drop_index("ix__annotation_user_id", table_name="annotation")
+    op.drop_constraint(
+        "fk__annotation__user_id__user", "annotation", type_="foreignkey"
+    )
+    op.drop_column("annotation", "user_id")
+
+
+def downgrade():
```

Comment:
I don't think we need to keep copying the two commands in reverse.

We could in theory import the previous migration and swap the steps but not sure how safe importing migrations is long term.